app/assets/javascripts/application.js

```diff
@@ -76,7 +76,7 @@ function onContentLoad() {
     // required in case the link was "enabled" after the
     // function has registered.
 
-    return !this.disabled;
+    return !$(this).attr('disabled','disabled');
```

Comment:
This will set the `disabled` attribute to `disabled`, not return whether it is disabled (https://api.jquery.com/attr/#attr2). Also, `disabled` should be handled using `.prop`, not `.attr`.

Reply:
sorry I am not strong with jQuery,
I saw some example with this syntax and used it.
`.prop` seem to return `undefined` for some reason, so lets use:
`!$(this).attr('disabled') === 'disabled';` instead